File: trunk/Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/ProfileManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IProfileManager
	'	Inherits IManagerBase(Of Profile, System.Int32)
	'End Interface

	Partial Class ProfileManager
		Inherits ManagerBase(Of Profile, System.Int32)
		Implements IProfileManager
	End Class
End Namespace

File: Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/ProfileManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IProfileManager : IManagerBase<Profile, int>
    {
	}
	
	partial class ProfileManager : ManagerBase<Profile, int>, IProfileManager
    {
	}
}